src/main/kotlin/fi/metatavu/lipsanen/projects/milestones/tasks/proposals/ChangeProposalController.kt

```diff
@@ -175,6 +181,27 @@ class ChangeProposalController {
         return persist(foundProposal)
     }
 
+    /**
+     * Create notifications about proposal status updates
+     *
+     * @param proposal proposal
+     * @param userId user id
+     * @param newStatus new status
+     */
+    private suspend fun notifyProposalUpdateStatus(
+        proposal: ChangeProposalEntity,
+        userId: UUID,
+        newStatus: ChangeProposalStatus
+    ) {
```

Comment:
I am wondering if it is a good idea to notify also about a new change proposal created?

Reply:
Notify the admins + assignees of the task that got the proposal?

Reply:
Done, added new type in specs + here